src/realm/error_codes.cpp

```diff
@@ -109,15 +109,15 @@ class CloseStatusErrorCategory : public std::error_category {
 
 } // unnamed namespace
 
-const std::error_category& websocket_close_status_category() noexcept
+const std::error_category& close_status_category() noexcept
 {
-    static const CloseStatusErrorCategory category = {};
+    static const StatusErrorCategory category = {};
     return category;
 }
 
 std::error_code make_error_code(ErrorCodes::Error error) noexcept
 {
-    return std::error_code{error, websocket_close_status_category()};
+    return std::error_code{error, close_status_category()};
```

Comment:
Should we use `close_status_category()` for all `std::error_codes` created from a `Status`?

Reply:
Oh - we'll use a `close_status_category()` for just websocket errors and maybe others for other types of status errors... or error_categories will just go away with the error unification changes 🤞

Reply:
exactly